### PR TITLE
How to render a content in a node

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: en/user_guide/content_display.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+--------------------------
+The configurable content block renders a single content, chosen by the contributor. A typical use is
+to create a focus on a specific content somewhere in the page, that could not be the principal
+information of the page, such has a mini-block on the last product developped by the company, or the
+employee of the month.
+
+Contribution
+````````````
+To get the block working, the contributor must first choose a content type then choose the content
+type to display.
```
Comment (itkg-bfouche) [inline en/user_guide/content_display.rst]: choose a content type then choose the content to display?

File: en/user_guide/content_display.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+block. A typical use may be to display a list of short previews of several articles, or a listing of
+customers.
+
+Contribution
+````````````
+Besides the common settings for all blocks, the content list block requires two specific types of
+settings:
+* Criterias: These are a combination of two basic criterias. The contributor must choose a content
+type and/or keyword(s) to trigger the contents.
+* Rendering: The contributor has two ways to render this block. He can use the standard template of
```
Comment (itkg-bfouche) [inline en/user_guide/content_display.rst:27]: typo error: the bullet list doesn't display correctly

**Comment by ngilain** [comment]
@itkg-bfouche , @nicolasThal , @itkg-rboessel , @itkg-nanne , @itkg-alavieille

File: en/user_guide/content_display.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Contribution
+````````````
+Besides the common settings for all blocks, the content list block requires two specific types of
+settings:
+
+* Criterias: These are a combination of two basic criterias. The contributor must choose a content
+  type and/or keyword(s) to trigger the contents.
+
+* Rendering: The contributor has two ways to render this block. He can use the standard template of
+  the content list block or create one in the Back Office to be used in the context of that node. To
```
Comment (veolys-nanne) [inline en/user_guide/content_display.rst]: context of that block? If you've got several contents list block in this node.

File: en/user_guide/content_display.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  use the standard template, put the template usage on Off. To use a unique node template, put the
+  template usage on On, then use the rich text below to create the template. You can use the twig syntax.
+  This template will be used only for that block in that node. Other content list blocks in that node
+  or in others are not been concerned by that template.
+
+
+Content block
+-------------
+The content block renders the content whose id is given in the address of the page. A typical use is
```
Comment (veolys-nanne) [inline en/user_guide/content_display.rst]: Maybe a link to explain this id ie contentId

File: en/user_guide/content_display.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Contribution
+````````````
+Besides the common settings for all blocks, the content list block requires two specific types of
+settings:
+
+* Criterias: These are a combination of two basic criterias. The contributor must choose a content
+  type and/or keyword(s) to trigger the contents.
+
+* Rendering: The contributor has two ways to render this block. He can use the standard template of
```
Comment (nicolasThal) [inline en/user_guide/content_display.rst:27]: Can you put sub entry for the different way?

File: en/user_guide/content_display.rst (hunk 4 above)
Comment (nicolasThal) [inline en/user_guide/content_display.rst]: `in the page address`

File: en/user_guide/content_display.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  template usage on On, then use the rich text below to create the template. You can use the twig syntax.
+  This template will be used only for that block in that node. Other content list blocks in that node
+  or in others are not been concerned by that template.
+
+
+Content block
+-------------
+The content block renders the content whose id is given in the address of the page. A typical use is
+to create a generic node rendering specific content detailed information. By varying the content id
+in the url, the same node can be used to display various articles or customer details. For instance
```
Comment (nicolasThal) [inline en/user_guide/content_display.rst]: Can you make a new paragraph for the example?

File: en/user_guide/content_display.rst [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+a node with the pattern /customer/{contentId} would be accessed in Front Office via url /customer/
+paul-smith and then the content block would display information about Paul Smith. It could also be
+accessed via url /customer/john-anderson, and then displaying information about John Anderson.
+
+Contribution
+````````````
+On this block, like on the content list block, the contibutor can use the standard template of the
+block or create a specific one for that block/node using the rich text.
+
+**To get this block working, it's important to set the node pattern in a specific way. As the content
```
Comment (nicolasThal) [inline en/user_guide/content_display.rst]: node urlPattern

File: en/user_guide/content_display.rst [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+id comes from the url, the node pattern must include the string {contentId} or the content id will
+not be available**
+
+
+Configurable content block
+--------------------------
+The configurable content block renders a single content, chosen by the contributor. A typical use is
+to create a focus on a specific content somewhere in the page, that could not be the principal
+information of the page, such has a mini-block on the last product developped by the company, or the
+employee of the month.
```
Comment (veolys-nanne) [inline en/user_guide/content_display.rst]: Theses two examples are contents list reduce to one entry. ^^

File: en/user_guide/content_display.rst (hunk 7 above)
Comment (nicolasThal) [inline en/user_guide/content_display.rst]: Do not put the word important, the emphasis is already stating this fact

File: en/user_guide/content_display.rst [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+block or create a specific one for that block/node using the rich text.
+
+**To get this block working, it's important to set the node pattern in a specific way. As the content
+id comes from the url, the node pattern must include the string {contentId} or the content id will
+not be available**
+
+
+Configurable content block
+--------------------------
+The configurable content block renders a single content, chosen by the contributor. A typical use is
```
Comment (nicolasThal) [inline en/user_guide/content_display.rst]: Can you also put the example in a different paragraph?

File: en/user_guide/content_display.rst (hunk 8 above)
Comment (ngilain) [inline en/user_guide/content_display.rst]: Hum it could be yes. But for now it's hard to trigger a specific content with content list block. Plus as the name says, it should be used to display a list, not a single content. Maybe with something visual to separate each content that would make no sense in the context of a single content.

**Comment by ngilain** [comment]
@nicolasThal , @itkg-nanne : update done

File: en/user_guide/content_display.rst [hunk 10]
```diff
@@ -0,0 +1,76 @@
+Contents display
+================
+
+Once contributed, there is only one way to display a content: using content blocks in nodes. Open
+Orchestra comes with 3 block types able to render a content, but a developper can create his own block
+to match its need. The 3 standard Open Orchestra content blocks are:
```
Comment (juchi) [inline en/user_guide/content_display.rst]: his needs.
small numbers (1 digit) are usually written in letters

File: en/user_guide/content_display.rst [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+* Content list block
+* Content block
+* Configurable content block
+
+
+Content list block
+------------------
+This block renders in Front Office a list of contents matching criterias given by the setting of the
```
Comment (juchi) [inline en/user_guide/content_display.rst]: a criterion / criteria

File: en/user_guide/content_display.rst [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+block. A typical use may be to display a list of short previews of several articles, or a listing of
```
Comment (juchi) [inline en/user_guide/content_display.rst]: A bit heavy to read, i would rephrase to something like "a typical use case may be the display of short previews for an article list"

File: en/user_guide/content_display.rst [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+customers.
+
+Contribution
+````````````
+Besides the common settings for all blocks, the content list block requires two specific types of
+settings:
+
+* Criterias: These are a combination of two basic criterias. The contributor must choose a content
```
Comment (juchi) [inline en/user_guide/content_display.rst]: criteria

File: en/user_guide/content_display.rst [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+  type and/or keyword(s) to trigger the contents.
+
+* Rendering: The contributor has two ways to render this block. He can use the standard template of
+  the content list block or create one in the Back Office to be used in that specific context, ie only
```
Comment (juchi) [inline en/user_guide/content_display.rst]: i.e.

File: en/user_guide/content_display.rst [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+  for that instance of block in that node.
```
Comment (juchi) [inline en/user_guide/content_display.rst]: that block instance

File: en/user_guide/content_display.rst [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+Using the standard template
+'''''''''''''''''''''''''''
+To use the standard template, put the template usage on Off.
+
+Using a "one shot" template
+'''''''''''''''''''''''''''
+To use a new specific and unique template, put the template usage on On, then use the rich text below
```
Comment (juchi) [inline en/user_guide/content_display.rst]: i think you mean "rich text area"

File: en/user_guide/content_display.rst [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+
+
+Content block
+-------------
+The content block renders the content whose id is given in the page address. A typical use is to
+create a generic node rendering specific content detailed information. By varying the content id
+in the url, the same node can be used to display various articles or customer details.
+
+For instance a node with the pattern /customer/{contentId} would be accessed in Front Office via
+url /customer/paul-smith and then the content block would display information about Paul Smith. It
```
Comment (juchi) [inline en/user_guide/content_display.rst]: would display some information

File: en/user_guide/content_display.rst [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+Anderson.
+
+Contribution
+````````````
+On this block, like on the content list block, the contibutor can use the standard template of the
+block or create a specific one for that block/node using the rich text.
+
+**To get this block working, the node url pattern has to be set in a specific way. As the content
+id comes from the url, the node pattern must include the string {contentId} or the content id will
+not be available** For more information about this, see `node configuration`_.
```
Comment (juchi) [inline en/user_guide/content_display.rst]: A dot is missing after "available"

File: en/user_guide/content_display.rst [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+
+
+Configurable content block
+--------------------------
+The configurable content block renders a single content, chosen by the contributor.
+
+A typical use is to create a focus on a specific content somewhere in the page, that could not be
```
Comment (juchi) [inline en/user_guide/content_display.rst]: "A typical use case"
However, this expression was already used above.

**Comment by ngilain** [comment]
@juchi : update done

**Comment by juchi** [comment]
:+1:
